src/pages/home/Home.tsx

```diff
@@ -1,88 +1,130 @@
-import React, { useMemo } from 'react';
+import React from 'react';
 import {
   IonContent,
   IonPage,
   IonListHeader,
   IonGrid,
   IonRow,
   IonCol,
-  IonCard,
-  IonCardHeader,
-  IonCardTitle,
-  IonCardContent
+  IonSearchbar,
+  IonItem,
+  IonRouterLink,
+  IonButton,
+  IonList,
+  IonLabel,
+  IonSegment,
+  IonSegmentButton,
+  IonIcon,
+  IonThumbnail
 } from '@ionic/react';
-import data from "../../assets/data/data.json";
+import { people, document, documents } from 'ionicons/icons';
 import './Home.css';
-import parse from 'html-react-parser';
+import data from "../../assets/data/data.json";
 
 const Home: React.FC = () => {
-
-  const moemorizedCases = useMemo(() =>  data.cases, []);
-  const moemorizedTopics = useMemo(() =>  data.topics, []);
+  const guides = data.topics;
 
   return (
     <IonPage>
       <IonContent>
         <IonListHeader>
```

Comment:
Move these blocks of content out of `IonList` -- they're not list items and it's messing with the padding. Then IonListHeader will fall away too and you can simply use `<h2>`. You may need to put `className="ion-padding"` on the `IonContent`.